aws/resource_aws_codebuild_project_test.go

```diff
@@ -459,6 +464,11 @@ resource "aws_codebuild_project" "foo" {
     type = "NO_ARTIFACTS"
   }
 
+  cache {
```

Comment:
Rather than modifying the baseline configuration, we should probably create another acceptance test and configuration(s) that implements these steps:
* Create project without the `cache` configuration
* Adding the `cache` configuration to the project
* Updates the location
* Removes the `cache` configuration from the project